packages/npm-dependencies-validation/src/api.ts

```diff
@@ -0,0 +1 @@
+export { getDependencyIssues } from "./index";
```

Comment:
you may also want to export relevant types here (e.g used in the `getDependencyIssues` signature)

Reply:
Done